.github/workflows/deploy-storybook.yml

```diff
@@ -35,10 +35,10 @@ jobs:
 
       - name: Create Release
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}
```

Comment:
FYI on [the secrets page](https://github.com/virtru/virtuoso-design-system/settings/secrets), this is `GH_TOKEN`